pingouin/multivariate.py

```diff
@@ -254,3 +254,102 @@ def multivariate_ttest(X, Y=None, paired=False):
     stats = {'T2': t2, 'F': fval, 'df1': df1, 'df2': df2, 'pval': pval}
     stats = pd.DataFrame(stats, index=['hotelling'])
     return _postprocess_dataframe(stats)
+
+def box_m (data, vars=None, group=None, alpha=.001):
```

Comment:
@GanshengT  For consistenty with Pingouin other functions, can you please rename `vars` to `dvs` (= dependent variables)